doc/releases/migration-guide-4.0.rst

```diff
@@ -51,6 +51,11 @@ Mbed TLS
   * ``CONFIG_MBEDTLS_ENTROPY_ENABLED`` is now :kconfig:option:``CONFIG_MBEDTLS_ENTROPY_C``,
   * ``CONFIG_MBEDTLS_ZEPHYR_ENTROPY`` is now :kconfig:option:``CONFIG_MBEDTLS_ENTROPY_POLL_ZEPHYR``.
 
+* The Kconfig option ``CONFIG_MBEDTLS_SSL_EXPORT_KEYS`` was removed because the
+  corresponding build symbol was removed in Mbed TLS 3.1.0. Starting from this
+  release ``MBEDTLS_SSL_EXPORT_KEYS`` is assumed to be enabled in Mbed TLS.
```

Comment:
Maybe this would be cleared:
```suggestion
  corresponding build symbol was removed in Mbed TLS 3.1.0 and is now assumed to be enabled.
```
Because now reading this, I thought that `Starting from this release` could refer to `this release` of Zephyr, not the Mbed TLs one you are referring to.